hubbard/hamiltonian.py

```diff
@@ -82,7 +103,12 @@ def __init__(self, TBHam, n=0, U=0.0, q=(0., 0.), nkpt=[1, 1, 1], kT=1e-5, units
             else:
                 self.q[0] = ntot / 2
 
-        self.sites = len(self.geometry)
+        self.q0 = np.zeros((self.sites))
```

Comment:
Why double parentheses?

Reply:
I thought it was needed but I see that is not, thanks!